sms-notifier/LambdaFunction.js

```diff
@@ -2,30 +2,28 @@
 var AWS = require('aws-sdk');
 var sns = new AWS.SNS();
 
-// define configuration
-const phoneNumber =''; // Insert phone number here. For example, a U.S. phone number in E.164 format would appear as +1XXX5550100
-
 //main function which gets AWS Health data from Cloudwatch event
 exports.handler = (event, context, callback) => {
+    //get phone number from Env Variable
+    var phoneNumber = process.env.PHONE_NUMBER;
     //extract details from Cloudwatch event
     eventName = event.detail.eventTypeCode
     healthMessage = 'The following AWS Health event type has occured: ' + eventName + ' For more details, please see https://phd.aws.amazon.com/phd/home?region=us-east-1#/dashboard/open-issues';
```

Comment:
pputhran: The above could be rewritten as
healthMessage = `The following AWS Health event type has occured: - ${eventName}. For more details, please see https://phd.aws.amazon.com/phd/home?region=us-east-1#/dashboard/open-issues`; */